app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -3252,6 +3253,24 @@ class PluginApi {
     addLegacyAboutPageStat(name);
   }
 
+  /**
+   * Registers a component class that will be rendered within the AdminPageHeader component
+   * only on plugins using the AdminPluginConfigPage and the new plugin "show" route.
+   *
+   * This component will be passed an `@actions` argument, with Primary, Default, Danger,
+   * and Wrapped keys, which can be used for various different types of buttons (Wrapped
+   * should be used only in very rare scenarios).
+   *
+   * This component would be used for actions that should be present on the entire UI
+   * for that plugin -- one example is "Create export" for chat.
+   *
+   * @param {string} pluginId - The `dasherizedName` of the plugin using this component.
+   * @param {Class} componentKlass - The JS class of the component to render.
```

Comment:
It could be called `componentClass`, right?

Reply:
You're right...I guess I just have some weird habit of always using Klass for these kind of things. Will change :+1: